package.json

```diff
@@ -12,6 +12,16 @@
   "module": "./dist/index.modern.js",
   "react-native": "./dist/index.modern.js",
   "types": "./dist/src/index.d.ts",
+  "exports": {
+    "./package.json": "./package.json",
+    "./": "./",
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "module": "./dist/index.modern.js",
+      "import": "./dist/index.modern.js",
+      "default": "./dist/index.umd.js"
```

Comment:
shouldn't it be `"require": "./`?

https://nodejs.org/api/packages.html#packages_conditional_exports